### DISCUSSION TITLE
[Feature] Skip, overwrite, let me decide on duplicate pictures uploaded from mobile

### DISCUSSION THREAD

**Discussion by mellow65** [immich-app/immich#3770] [Feature Request]
### The feature

I went through the discussions and didn't find anything exact to what I'm looking at here, I noticed when I upload pictures from my phone that were already uploaded from the web it would still upload them and add a +1 on the end of it.   The pictures have the same name and data associated with it, but they are different size due to one being pulled from google photos (space saver).  I'm assuming Immich is looking at all of the files attributes, not just the name and date to make the decision if it's a duplicate or not. 

Thanks

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by Jasmin25** [comment]
+1 for this feature, this seems to be a common occurrence for me too!

**Comment by bo0tzz** [comment]
The current duplicate detection is purely based on whether the files are identical. I'm closing this as a duplicate of #1968
